orchagent/fdborch.cpp

```diff
@@ -63,3 +70,218 @@ bool FdbOrch::getPort(const MacAddress& mac, uint16_t vlan, Port& port)
 
     return true;
 }
+
+void FdbOrch::doTask(Consumer& consumer)
+{
+    SWSS_LOG_ENTER();
+
+    auto it = consumer.m_toSync.begin();
+    while (it != consumer.m_toSync.end())
+    {
+        KeyOpFieldsValuesTuple t = it->second;
+
+        string key = kfvKey(t);
+        string op = kfvOp(t);
+
+        FdbEntry entry;
+        if (!splitKey(key, entry))
+        {
+            it = consumer.m_toSync.erase(it);
+            continue;
+        }
+
+        if (op == SET_COMMAND)
+        {
+            string port;
+            string type;
+            bool port_defined = false;
+            bool type_defined = false;
+
+            for (auto i : kfvFieldsValues(t))
```

Comment:
can you add fdb app db schema to https://github.com/Azure/sonic-swss/blob/master/doc/swss-schema.md